### DISCUSSION TITLE
Keyball 44

### DISCUSSION THREAD

**Discussion by R4vn0s** [Yowkees/keyball#592] [Q&A]
I'm in the process of assembly a Keyball44, I've soldered all for the trackball side, so I flashed the pro micro type C and started testing, all the letters and LEDs work fine, but the sensor for trackball doesn't work, I resoldered everything related but still don't work. Does the sensor need the other keyboard side to be connected? if not,  is there a way to test the sensor to check if is ok?

**Comment by Ryo624** [comment]
The trackball sensor can be checked without connecting the other keyboard.
However, we will not be able to confirm if the trackball sensor is normal until after we have installed the L-shaped conthrough pin on the board, so we will have to finish the assembly completely to the end and write the Keyball44_via firmware to the board.

**Comment by R4vn0s** [comment]
![PXL_20240527_204635795](https://github.com/Yowkees/keyball/assets/42302827/8dccce6a-b076-46a5-ac18-5f21112e72ac)
![PXL_20240527_204645553](https://github.com/Yowkees/keyball/assets/42302827/f78807d1-d56e-48f0-92ea-ec57e0d6f43a)
![PXL_20240527_204709482](https://github.com/Yowkees/keyball/assets/42302827/ed774734-f7e3-429d-ac23-7cea6ee15ea3)
![PXL_20240527_204800615](https://github.com/Yowkees/keyball/assets/42302827/bbff0d5c-360f-4ba1-b909-7d5a6895460e)
![PXL_20240527_204831778](https://github.com/Yowkees/keyball/assets/42302827/e0894ae1-4cee-4040-87c5-c66bd62d365c)
![PXL_20240527_204954534](https://github.com/Yowkees/keyball/assets/42302827/2377b93d-d589-4db9-87cd-18e847b094db)
![PXL_20240527_205024440](https://github.com/Yowkees/keyball/assets/42302827/de6ac8b6-b6ff-45cc-9075-f6ec65fc59bd)


I have already soldered everything in the right side(the one with the trackball),  flashed the pro micro, and tested the keys, all the key and leds work fine, but the sensor doesn't work

**Comment by Ryo624** [comment]
Try re-soldering the L-shaped conthrough pin and the trackball board.

May I also check the soldering of the ProMicro on the back side of the board if possible?

**Comment by R4vn0s** [comment]
Hi @Ryo624, thanks a lot for your time, I resoldered the L - Shaped pin again, also de pro-micro and now it is working, thanks a lot.

Now I must wait until arrive my second pro micro, I was a dumb and bought only one. ありがとう

**Comment by Ryo624** [comment]
I am glad to hear that it is now working safely.

Let me know if you need any more help.
